### ISSUE TITLE
brave_installer-delta-x64.exe - Flagged/Blocked as PUA by AV engine (Sophos)

### ISSUE THREAD

**Issue by RonnyTNL** [brave/brave-browser#32361]
## Description 
Inline Update of Brave browser failed because the non-code-signed brave_installer-delta-x64.exe is blocked by the AV engine detected as PUA.

## Steps to Reproduce
Use Brave browser with Sophos Home (Premium) or HitmanPro.Alert installed and wait till the updater kicks in.
You will get an AV blocked PUA detection.

## Actual result:
Update broken

## Expected result:
Update finished flawlessly

## Reproduces how often: 
100% on latest delta installer SHA2: fcd8e71ae0764c17530ae3dcbcb2a8c8f9f9d785ba51ef6639b87e6763bbee32

https://www.virustotal.com/gui/file/fcd8e71ae0764c17530ae3dcbcb2a8c8f9f9d785ba51ef6639b87e6763bbee32/details

## Brave version (brave://version info)
Windows 10 x64 - Brave x64 update from version Version 1.57.47 -> 1.57.49

## Version/Channel Information:
N/A

- Can you reproduce this issue with the current release? 
- Can you reproduce this issue with the beta channel? 
- Can you reproduce this issue with the nightly channel? 

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? - NO
- Does the issue resolve itself when disabling Brave Rewards?  - NO
- Is the issue reproducible on the latest version of Chrome?  - Not tried by likely as FP has potential to flag all upcoming delta installers.

## Miscellaneous Information:
Might want to code-sign the delta installers so this doesn't happen in the future.

**Comment by RonnyTNL** [comment]
Also reported the potential FP to Sophos.
Can't find an easy way to report to Trapmine, but their ML engine also seems unhappy with some characteristics of this code

**Comment by kirchwitz** [comment]
Can verify this exact same problem here (Sophos, Windows 10). Never had any such problems in the past, Brave updates always worked like a charm. Now I'm curious what the explanation is because my IT departments asks me what strange things I'm doing there that trigger the alarm... :-)

**Comment by RonnyTNL** [comment]
> Can verify this exact same problem here (Sophos, Windows 10). Never had any such problems in the past, Brave updates always worked like a charm. Now I'm curious what the explanation is because my IT department asks me what strange things I'm doing there that trigger the alarm... :-)

Sophos team confirmed FP, just not sure if this would resurface on a newer version of the delta-installer.

**Comment by RonnyTNL** [comment]
brave_installer-delta-ia32.exe is also affected by this 

5 ML engines flag this one;
https://www.virustotal.com/gui/file/b4827c0f1a2090d4e512e419de0570d3b1e89e0fdd0ec09c5723441c0ee47d16/detection

/cc @fmarier

**Comment by mherrmann** [comment]
Thank you for reporting this and sorry for the trouble. It's strange that it used to work in the past and only becomes a problem now. Maybe Sophos changed something.

The issue has already been fixed in #31977. But it is currently only available in Brave Nightly. I'll look into getting it uplifted to Beta and Release.

**Comment by mherrmann** [comment]
I will close this issue (essentially because it is a duplicate of #31977). You can track the progress of it getting fixed in Brave Beta and Brave Release in the following PRs:

- https://github.com/brave/brave-core/pull/19781
- https://github.com/brave/brave-core/pull/19782

**Comment by kjozwiak** [comment]
Labelling as `QA/No` after discussing with @mherrmann. QA will run through verifications using https://github.com/brave/brave-browser/issues/31977.
